### PR TITLE
Polish the 2.098 changelog

### PR DESCRIPTION
**Pull request by dkorpel**
- Remove the markdown `#` from the first line (it's rendered as an actual `#` char)
- Clarify / update some information

### CODE REVIEW

File: changelog/ImportC.dd [hunk 1]
```diff
@@ -191,10 +191,6 @@ is advisable.
 * Tag symbols are part of the global symbol table, not the special tag
 symbol table like they are supposed to reside in.
 
-* Octal literals are not supported.
-
-* Integer literal suffixes are not correct.
```
Comment (dkorpel) [inline changelog/ImportC.dd:196]: Fixed by https://github.com/dlang/dmd/pull/12539

**Comment by kinke** [comment]
Should target stable. :)

**Comment by dkorpel** [comment]
Stable doesn't have https://github.com/dlang/dmd/pull/13023 and https://github.com/dlang/dmd/pull/13054, so it's giving conflicts.

**Comment by kinke** [comment]
Oh I assumed the PR title was correct. ;)

**Comment by dkorpel** [comment]
Huh, is it not 2.098? :sweat_smile:

**Comment by kinke** [comment]
Unless I'm totally mistaken, master is very early 2.099, stable is 2.098.

**Comment by dkorpel** [comment]
I see, though that makes my wonder why the changelog entries from 2.098 aren't deleted on master by now.

**Review by RazvanN7 — APPROVED** [review]

**Comment by dkorpel** [comment]
Should I make a follow up Pull Request to transfer the changes to stable and delete the entries in master?

**Pull request merged**